### DISCUSSION TITLE
What are the units in Dualsphysics? and the values of the rheological parameters taken in the HBP model

### DISCUSSION THREAD

**Discussion by gggggakki** [DualSPHysics/DualSPHysics#247] [DUALSPHYSICS_V5.2]
1. i would like to know what are the base units in Dualsphysics? For example, does the 1 in <drawbox> stand for 1 meter?
2. When I substitute the rheological parameters measured by the rheometer into the HBP model, the phenomenon presented does not match the actual one at all (the simulated fluid seems to be a solid), and the simulated flow phenomenon seems to be normal after I reduce the parameters by two to three orders of magnitude. So I'm wondering if this is due to the size of the simulation? (The simulated dimensions are the same as the physical test dimensions, which are only 30cm-40cm in size, so the dimensions I set in Dualsphysics are only 0.3-0.4) or is it something else. I hope someone can answer this question, thanks a lot!

**Comment by alexbexe** [comment]
1) we use the International System of Units
2) please contact Dr Georgios Fourtakas for questions related to that approach

Regards
Alex

**Comment by gfourtakas** [comment]
We use the "specific" parameters, divided by the desnity thus the 2-3 orders of magnitude difference, on the xml you can find the units
Let me know how you are getting on after you modify your values

**Reply by sopRSH** [reply]
Dear Dr. Fourtakas,

I am currently setting up simulations in DualSPHysics with non-Newtonian, shear-thinning fluids, and I have a question regarding the "visco" parameter in the XML input file.
I have two sets of rheological data with consistency coefficients:
K = 0.3 (Pa·s)^n
K = 7 (Pa·s)^n
I divided these values by the fluid density (ρ = 1800 kg/m³) and entered them as the "visco" parameter. However, in the simulation results, the behavior of both fluids appears very similar, almost identical.
What are the correct units for the visco parameter in the XML?
Should the value be divided by the density, or should it be entered directly in Pa·s^n?
I deeply need your guidance on this matter and would greatly appreciate your response.
Best regards,
